manager/integration/tests/test_scheduling.py

```diff
@@ -899,8 +895,10 @@ def test_data_locality_basic(client, core_api, volume_name, pod, settings_reset)
 
     volume4 = volume4.updateDataLocality(dataLocality="disabled")
     volume4 = volume4.updateReplicaCount(replicaCount=2)
-    wait_for_volume_healthy(client, volume4_name)
+    wait_for_rebuild_start(client, volume4_name)
+    wait_for_rebuild_complete(client, volume4_name)
     volume4 = wait_for_volume_replica_count(client, volume4_name, 2)
+    volume4 = wait_for_volume_healthy(client, volume4_name)
```

Comment:
Must wait for the volume to rebuild, and then healthy because if we don't, the volume might be in the state with 1 failed replica and 1 running replica.